custom_components/kia_uvo/HyundaiBlueLinkAPIUSA.py

```diff
@@ -106,13 +107,20 @@ def get_cached_vehicle_status(self, token: Token):
         url = self.API_URL + "rcs/rvs/vehicleStatus"
         headers = self.API_HEADERS
         headers["accessToken"] = token.access_token
+        headers["vin"] = token.vehicle_id
+        headers["refresh"] = "false"
```

Comment:
Could we try move this to the api_headers?  This will let you not need to repeat it once the remaining methods to be built out.  It will depend if it breaks login or not.